client/app/pods/components/question-multi-radio/component.js

```diff
@@ -0,0 +1,9 @@
+import QuestionComponent from 'tahi/pods/components/question/component';
+
+export default QuestionComponent.extend({
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.